Домашка/Program.cs

```diff
@@ -1,12 +1,29 @@
 ﻿using System;
 
-namespace Домашка
+namespace Трапеция
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+
+            Console.WriteLine("введите высоту");
+            var h = double.Parse(Console.ReadLine());
+            Console.WriteLine("введите большее основание");
+            var b = double.Parse(Console.ReadLine());
+            Console.WriteLine("введите меньшее основание");
+            var c = double.Parse(Console.ReadLine());
+            double s = h * (c + b) / 2;
+            Console.WriteLine("Площадь трапеции равна = ");
+            Console.WriteLine(s);
+            double a = (b - c) / 2;
+            double d = Math.Sqrt((a * a) - (h * h));
```

Comment:
Все таки нашел неправильную формулу. Теорема Пифагора: "квадрат гипотенузы равен сумме квадратов катетов".
```suggestion
            double d = Math.Sqrt((a * a) + (h * h));
```